README.md

```diff
@@ -1 +1,161 @@
-# ros-frontend
\ No newline at end of file
+# Resource Optimisation Service (Frontend)
+
+Resource Optimization is a service aimed to provide existing Red Hat Enterprise Linux customers with an ability to grasp control over their investment in the Public Cloud. Read more about it [here](https://www.redhat.com/en/blog/optimize-public-cloud-workloads-rhel-red-hat-insights-resource-optimization) and go though the confluence section(check below section) for more details on same.
+
+
+# Getting Started:
+
+Please refer [Resource Optimization Service Confluence](https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=ROS&title=Resource+Optimization+Service+Home) (requires VPN)) page to find all documents related to the project
+
+# Prerequisites
+
+### Install
+
+- [Node](https://nodejs.org/en/download/)
+- [Docker](https://docs.docker.com/get-docker/)
+
+### Make sure to have test account created in CI, QA, Staging envrionments. For more details please check [here](https://docs.google.com/presentation/d/1XcbAplLOerPXaEF8DYgS336xwdfUuiUPXGXtuBfk1oY/edit?usp=sharing)
```

Comment:
Ditto. This is an internal/private link too.

Reply:
Yes, this step should be present in internal documentation.